src/Kestrel.Core/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -8998,4 +8998,235 @@ public bool MoveNext()
             }
         }
     }
+
+    public partial class HttpResponseTrailers
+    {
+        private static byte[] _headerBytes = new byte[]
+        {
+            13,10,69,84,97,103,58,32,
```

Comment:
What's this used for?

Reply:
Generated magic 😁 `\r\nETag: `. Not needed for HTTP/2 but would be needed for an eventual HTTP/1.1 implementation of trailers.